prime-router/metadata/fhir_mapping/hl7/datatype/Identifier_var.yml

```diff
@@ -45,8 +45,19 @@ period:
     start: $start
     end: $end
 
-# For OIDs
 extension_1:
```

Comment:
Added this extension to allow for `namespaceId` to be used as an alternative to `systemCX` without removing `systemCX` altogether. Helps us match the EI mapping [here](https://docs.google.com/spreadsheets/d/1pKB8A1ZmR2uym9H5Isdh8VNvO30t61xpjHBjaUKGaio/edit#gid=0). See `identifier_2a` in `ServiceRequest.yml` as an example.